code/_compile_options.dm

```diff
@@ -70,6 +70,12 @@
 #error You need version 514.1556 or higher
 #endif
 
+#if (DM_VERSION == 514 && DM_BUILD > 1571)
```

Comment:
this covers literally every new build

Reply:
yes, all new builds crash, until we get one that doesn't crash we gotta block 'em, 1571 is the last working number

Reply:
alright fikou confirmed 1575 works, i guess i fucked up testing that one